test/filesystem/file_nonmutating_test.rb

```diff
@@ -186,7 +186,10 @@ def test_blockdev?
   end
 
   def test_symlink?
-    assert_always_false(:symlink?)
+    zip_file = ::Zip::File.new('test/data/path_traversal/tuzovakaoff/symlink.zip')
+    assert(zip_file.file.symlink?('path'))
+    assert(!zip_file.file.symlink?('path/file.txt'))
```

Comment:
Please can you use refute here instead of negating the condition?

Reply:
Ah, ok. I haven't cleaned that one up yet 😬 

Leave it then and I'll sort the whole file out at some point. 

Cheers